public/styles/css/main.css

```diff
@@ -2,76 +2,126 @@
 *            Helpers           *
 ********************************/
 .italic {
-  font-style: italic;
-  color: #202020; }
+  font-style: italic; }
 
 .bold {
   font-weight: 700; }
 
 .sub-heading {
   font-size: 2.2rem; }
 
+.highlight {
+  color: #FF6F61;
+  font-weight: 700; }
+
+.size20 {
+  font-size: 20px; }
+
+.fas {
+  color: #FF6F61;
+  font-size: 2em;
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 15%;
+  margin-bottom: 10%; }
+
 /*******   END HELPERS   *******/
 body {
-  color: #3B3D3E;
-  background-color: #FFFEFD; }
+  color: #393D3F;
+  background-color: #FFF8F8;
+  margin: 10px 10px 10px 10px; }
 
 .container {
-  margin: 5px 10px; }
+  margin: 10px 10px 10px 10px; }
+
+/*******************************
+*         Landing Page         *
+********************************/
+header {
+  position: relative;
+  top: 29px; }
 
 h1 {
-  font-family: "Roboto", sans-serif;
-  color: #202020;
-  font-size: 3.5rem;
-  margin-top: 100px;
-  margin-left: 20px;
+  position: relative;
+  font-family: "Playfair Display", serif;
+  color: #393D3F;
+  font-size: 2.8rem;
+  letter-spacing: 0.1em;
+  margin-top: 40%;
+  margin-left: 4%;
```

Comment:
try writing the margin in one line? 
HINT: 
{ 
margin: [top][right][bottom][left] => 
margin: 10px 0 0 20px; 
}